HairSuite.Domain/Reservation.cs

```diff
@@ -67,7 +74,15 @@ public record ReservationId(Guid Value) : AggregateId(Value);
 
 public record HairdresserId(Guid Value);
 
-public record ReservationDate(DateTime Value);
+public record ReservationDate
+{
+    public DateTimeOffset Value { get; set; }
+
+    public ReservationDate(DateTimeOffset value)
+    {
+        Value = value.ToUniversalTime();
+    }
+}
```

Comment:
Is this update because the `Value` property is mandatory? If so, I believe you can come to the same result simply by setting the property as required.
``` c#
public record ReservationDate(required DateTime Value);
```

Reply:
Ahh. Missed that 👍🏼